exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/__init__.py

```diff
@@ -14,58 +14,6 @@
 
 
 """
-This library allows to export tracing data to an OTLP collector.
-
-Usage
------
-
-The **OTLP Span Exporter** allows to export `OpenTelemetry`_ traces to the
-`OTLP`_ collector.
-
-You can configure the exporter with the following environment variables:
-
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_TIMEOUT`
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_PROTOCOL`
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_HEADERS`
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_COMPRESSION`
-- :envvar:`OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE`
-- :envvar:`OTEL_EXPORTER_OTLP_TIMEOUT`
-- :envvar:`OTEL_EXPORTER_OTLP_PROTOCOL`
-- :envvar:`OTEL_EXPORTER_OTLP_HEADERS`
-- :envvar:`OTEL_EXPORTER_OTLP_ENDPOINT`
-- :envvar:`OTEL_EXPORTER_OTLP_COMPRESSION`
-- :envvar:`OTEL_EXPORTER_OTLP_CERTIFICATE`
```

Comment:
These are applicable to trace/metric/log exporters and that's why they are documented here. They should exist here.

Reply:
Apologies, missed where these were being defined within the parent dir!